incubator/hnc/test/e2e/issues_test.go

```diff
@@ -82,9 +83,12 @@ var _ = Describe("Issues", func() {
 
 	It("Should set SubnamespaceAnchorMissing condition if the anchor is missing - issue #501", func() {
 		// Setting up a 3-level tree with 'parent' as the root
-		MustRun("kubectl create ns", nsParent)
-		// create a subnamespace without anchor by creating a full namespace with SubnamespaceOf annotation
-		MustRun("kubectl create ns", nsSub1)
+		CreateNamespace(nsParent)
+
+		// create a subnamespace without anchor by crea
+		//ting a full namespace with SubnamespaceOf annotation
```

Comment:
Don't break a word ("crea" - "ting") with a newline. Better yet, just leave the existing comment as-is.

Reply:
Will do.